packages/cypress-internal/.npmrc

```diff
@@ -0,0 +1,2 @@
+@continuum:registry=https://npm.levelaccess.net/continuum/
+          //npm.levelaccess.net/continuum/:_authToken="AUTH_TOKEN"
```

Comment:
I think that it would be better to remove that file and add in documentation that it should be created when someone wants to use access continuum,